spec.html

```diff
@@ -12976,14 +12976,14 @@ <h1>
     <emu-clause id="sec-ordinarycreatefromconstructor" type="abstract operation">
       <h1>
         OrdinaryCreateFromConstructor (
-          _constructor_: a constructor,
+          _constructor_: a function object,
```

Comment:
why this change? "a constructor" is more correct and granular than "a function object".

Reply:
aha, thanks, you're correct.

Reply:
it's okay 👍🏻